src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -59,6 +59,8 @@
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
If we are only bringing in `guava` for this annotation, I would suggest removing the dependency and simply adding a method comment.